recipes/argo/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "argo" %}
+{% set version = "2.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: argo
+
+  url: https://github.com/argoproj/argo/releases/download/v{{ version }}/argo-darwin-amd64  # [osx]
+  sha256: 15dd9013654ea2a786d7eafc6f05e51e994ec2e7e595d84c27113a5f69eee918  # [osx]
```

Comment:
Please build from source, rather than repackaging binaries

Reply:
I am fine building from source but I can get the install files to go to PREFIX. I tried looking at other go libs without success. Could you help?

Reply:
I don't know much about go, sorry.  Hopefully @conda-forge/help-go can help.

Reply:
I tried multiple things still without success. Are you sure we can't build from official binaries? Those binaries are coming from the official repository. Here is an example of a golang conda forge package using the same strategy: https://github.com/conda-forge/go-spatial-feedstock/blob/master/recipe/meta.yaml#L10

Reply:
Yes, they may link to libraries we don't have on the system.

Reply:
Looking at the size of the binaries, they are probably all statically linked and self-sufficient.

Reply:
There is some weird cos6 vs cos7 issue with go that I have run into before

Reply:
I finally get it to compile on Linux and osx. Can we merge this and we can figure out windows later?

Reply:
Or maybe there is a way to call `build.sh` during Windows CI?

Reply:
Sorry, I just saw things. I'll help out

Reply:
You're a hero @sodre !